au4/src/processing/internal/processingactionscontroller.cpp

```diff
@@ -23,6 +31,51 @@ void ProcessingActionsController::notifyActionCheckedChanged(const ActionCode& a
     m_actionCheckedChanged.send(actionCode);
 }
 
+void ProcessingActionsController::clipCut()
+{
+    NOT_IMPLEMENTED;
+}
+
+void ProcessingActionsController::clipCopy()
+{
+    NOT_IMPLEMENTED;
+}
+
+void ProcessingActionsController::clipDelete()
```

Comment:
I think it's better to have two methods: 
1.
```
void ProcessingActionsController::clipDelete(const ActionData& args)
{
       IF_ASSERT_FAILED(args.count() > 0) {
               return;
       }

      processing::ClipKey clipKey = args.arg<processing::ClipKey >(0);
      ...
}
```
see https://github.com/igorkorsukov/audacity/blob/master/au4/src/projectscene/view/clipsview/clipcontextmenumodel.cpp#L14

2. void ProcessingActionsController::clipDeleteSelected()